app/javascript/components/AddItem.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addCar } from '../redux/Cars/carsSlice';
+
+const AddItem = () => {
```

Comment:
- [ ] OPTIONAL: Adding a car, and then going to the homepage seems to take a lot of time to render the cars.


![image](https://github.com/Kaghenimbale/car_collection/assets/57421823/38590690-e8ea-4fc3-b7d8-8133ebcf69f1)

![image](https://github.com/Kaghenimbale/car_collection/assets/57421823/4dc660dc-febd-47af-a7b5-f2a64728ad2e)

It takes about 20 seconds to finally load the updated list of cars from the server:

![image](https://github.com/Kaghenimbale/car_collection/assets/57421823/45d7d972-5c4f-4718-8071-e29cf8ed361d)

I suggest you solve this little problem, that way, the UX will be improved. 👍